iroha_client/tests/tx_rollback.rs

```diff
@@ -0,0 +1,96 @@
+#[cfg(test)]
+mod tests {
+    use async_std::task;
+    use iroha::{config::Configuration, isi, prelude::*};
+    use iroha_client::{
+        client::{self, Client},
+        config::Configuration as ClientConfiguration,
+    };
+    use std::{thread, time::Duration};
+    use tempfile::TempDir;
+
+    const CONFIGURATION_PATH: &str = "tests/test_config.json";
+
+    #[async_std::test]
+    //TODO: use cucumber to write `gherkin` instead of code.
+    async fn client_sends_transaction_with_invalid_instruction_should_not_see_any_changes() {
```

Comment:
Why is it called `tx_rollback`? I think it is simply that tx was invalid and therefore not executed or maybe I missed something

Reply:
I thought that it is appropriate term. We send transaction - it starts execution and rollback all changes if some of instructions are failed.
`In database technologies, a rollback is an operation which returns the database to some previous state.` - because we are not using database here we do not have commits and rollbacks as in database management systems, but I decided to use this terms for easier understanding by developers. What alternatives can you suggest?

Reply:
Ok, I see. Actually I missed the place where the actual `roll_back` feature was implemented, I thought it was only a new test. In fact the PR just has so many other changes :smile: . Now that I see the change, I agree that the rollback naming is okay.